newsfragments/789.feature

```diff
@@ -0,0 +1 @@
+Store static masks in FormatNXmx.py in a dict to avoid duplication
```

Comment:
since this is provided to users as part of the release notes suggest dict -> dictionary and emphasise that this is storing _unique_ masks